File: Model/Order.cs
using System;
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace bellatrix.Model
{
    public class Order
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }
        public DateTime DateCreated { get; set; }
        public string ClientName { get; set; }
        public string Description { get; set; }
        public decimal TotalPrice { get; set; }

        public override string ToString()
        {
            return $"{Id.Substring(0, 10)} - {DateCreated} - {ClientName} - {Description}: {TotalPrice:C}";
        }
    }
}

File: ClientApp/src/Model/Order.ts
import moment from "moment";
export default class Order {
  id: string;
  dateCreated: Date;
  clientName: string;
  description: string;
  totalPrice: number;

  constructor(
    id: string,
    dateCreated: Date,
    clientName: string,
    description: string,
    totalPrice: number
  ) {
    this.id = id;
    this.dateCreated = dateCreated;
    this.clientName = clientName;
    this.description = description;
    this.totalPrice = totalPrice;
  }

  dateCreatedFormatted(): string {
    return moment(this.dateCreated)
      .local()
      .format("YYYY-MM-DD HH:mm:ss");
  }

  totalPriceFormatted(): string {
    return `$${this.totalPrice}`;
  }

  toString() {
    return `${this.id} - ${this.dateCreatedFormatted()} - ${
      this.clientName
    } - ${this.description}: $${this.totalPrice}`;
  }
}
